functions/Get-DbaLinkedServer.ps1

```diff
@@ -0,0 +1,90 @@
+﻿function Get-DbaLinkedServer
+{
+<#
+.SYNOPSIS
+Gets all linked servers and summary of information from the sql servers listed
+
+.DESCRIPTION
+Retrieves information about each linked server on the instance
+
+.PARAMETER SqlServer
+The SQL Server that you're connecting to.
+
+.PARAMETER Credential
```

Comment:
Update the parameter name to `$SqlCredential` to match the parameters for the function.